dockstore-webservice/src/main/java/io/dockstore/webservice/core/Entry.java

```diff
@@ -272,4 +273,22 @@ public void update(S entry) {
      */
     @JsonIgnore
     public abstract Set<T> getVersions();
+
+    /**
+     *
+     * @param newDefaultVersion
+     * @return true if defaultVersion is a valid Docker tag
+     */
+    public boolean checkAndSetDefaultVersion(String newDefaultVersion) {
+        if (newDefaultVersion == null) {
```

Comment:
If the newDefaultVersion is null, do we really want to set it as null?

Reply:
That's a good point, null is really more of a no-op, I should change this